CRM/Financial/BAO/Payment.php

```diff
@@ -80,10 +79,7 @@ public static function create($params) {
     }
     if (!$fullyPaidPayLater) {
       $trxn = CRM_Contribute_BAO_Contribution::recordPartialPayment($contribution, $params);
-      $paid = CRM_Core_BAO_FinancialTrxn::getTotalPayments($params['contribution_id']);
-      $total = CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_Contribution', $params['contribution_id'], 'total_amount');
-      $cmp = bccomp($total, $paid, 5);
-      if ($cmp == 0 || $cmp == -1) {// If paid amount is greater or equal to total amount
+      if ($isPaymentCompletesContribution) {// If paid amount is greater or equal to total amount
```

Comment:
ug I  need to remove this inline comment next time I push a change (I don't want to kill current test run)